lucene/core/src/java/org/apache/lucene/codecs/lucene86/Lucene86PointsReader.java

```diff
@@ -93,18 +97,12 @@ public Lucene86PointsReader(SegmentReadState readState) throws IOException {
             BKDReader reader = new BKDReader(metaIn, indexIn, dataIn);
             readers.put(fieldNumber, reader);
           }
-          indexLength = metaIn.readLong();
-          dataLength = metaIn.readLong();
         } catch (Throwable t) {
           priorE = t;
         } finally {
           CodecUtil.checkFooter(metaIn, priorE);
         }
       }
-      // At this point, checksums of the meta file have been validated so we
```

Comment:
we don't lose safety, but in case of a corrupt meta file, it might be slightly more confusing in the sense that the suppressed exception will complain about a truncated index/data file